### ISSUE TITLE
Support for superscript and subscript in the text annotations.

### ISSUE THREAD

**Issue by suryasdan** [xournalpp/xournalpp#3975]
**Is your feature request related to a problem? Please describe.**
- I use xournal++ for technical writing including research and academic teaching purposes. This requires mathematical symbols involving superscripts and subscripts very often. I request the developers to add the feature of superscript and subscript in the text annotation tool.

**Describe the solution you'd like**
I want the text equivalent of $V_{GS}^2$ in latex. Such that "GS" should appear in the subscript and "2" in the superscript.

**Describe alternatives you've considered**
There are no alternatives, except using the latex tool and enlarging the image resulting out of it.

**Additional context**

**Comment by rolandlo** [comment]
An existing alternative would be to use unicode characters like
₀₁₂₃₄₅₆₇₈₉ for subscripts and
⁰¹²³⁴⁵⁶⁷⁸⁹ for superscripts.
A tool that helps is https://www.unicodeit.net/
See also [wikipedia](https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts) for what characters are supported.

**Comment by tattsan** [comment]
There are no capital subscripts in unicode. You may substitute small caps for the subscripts: 
https://en.wikipedia.org/wiki/Small_caps#Unicode

Your sample: `Vɢꜱ²`

See also #3468.

**Comment by suryasdan** [comment]
Is there any update on this? Inclusion of superscripts and subscripts to the text annotations in xournal++ will be an extremely useful feature. As I am a heavy user of Xournal++ (primarily on kubuntu), this feature will make all the academic, teaching and research oriented note-taking tasks, completely on xournal++ itself. Currently I am using the following workarounds:
1. Latex plugin: the issues are: (1) The fonts have to adjusted as the image created using latex plugin uses a font that is usually different than the text. (2) Latex plugin creates an image, which has to placed manually so as to appear as aligned within the text, especially for inline math formulas. FYI: Word processors like MS Office Word/Libreoffice (LO) writer do this kind of vertical alignment in their latest versions.
2. As rolandlo & tattsan rightly commented, unicode is definitely a good workaround, but the issues are: (1) Several characters are missing in the superscript and subscript table; (2) It is unnecessarily a burden on memory to remember all the unicode characters and/or keep referring to a spreadsheet (like LO Calc) just to type a few unicode symbols (as I often do); (3) Some of the unicode symbols do not align themselves with the remaining text.

In short, inclusion of the subscript/superscript functionality will enhance xournal++ almost to the standard of a word processor where one can type and annotate simultaneously on the same document. I am unaware of any free tool which does this. A million thanks to the developers and the team to making this wonderful software available as an opensource tool.
